node/node.go

```diff
@@ -843,9 +844,12 @@ func (app *App) initServices(ctx context.Context) error {
 		app.log.Panic("failed to create post setup manager: %v", err)
 	}
 
+	app.grpcPostService = grpcserver.NewPostService(app.log.Zap())
```

Comment:
How about using a dedicated post logger
```suggestion
	app.grpcPostService = grpcserver.NewPostService(app.addLogger(PostLogger, lg).Zap())
```